pkg/ocm/client.go

```diff
@@ -124,6 +124,10 @@ func (b *ClientBuilder) Build() (result *Client, err error) {
 	if b.cfg.URL != "" {
 		builder.URL(b.cfg.URL)
 	}
+	// when running ROSA against local CS, redirect AMS API calls to integration
```

Comment:
Not sure how exactly this works as I haven't taken a look at it. Should this update https://gitlab.cee.redhat.com/service/uhc-clusters-service#using-an-ocm-proxy instead of doing it in code?